react/features/toolbox/components/web/Toolbox.js

```diff
@@ -910,14 +917,13 @@ class Toolbox extends Component<Props> {
 
         this._setButtonsNotifyClickMode(buttons);
         const isHangupVisible = isToolbarButtonEnabled('hangup', _toolbarButtons);
-        const { order } = THRESHOLDS.find(({ width }) => _clientWidth > width)
-            || THRESHOLDS[THRESHOLDS.length - 1];
+        const thresholds = _isMobile ? THRESHOLDS_MOBILE_WEB : THRESHOLDS;
+        const { order } = thresholds.find(({ width }) => _clientWidth > width)
+            || thresholds[thresholds.length - 1];
         let sliceIndex = order.length + 2;
-
         const keys = Object.keys(buttons);
-
         const filtered = [
-            ...order.map(key => buttons[key]),
+            ...order.map(key => _.find(buttons, b => b?.key === key)),
```

Comment:
Not sure what we gain by this, but ok

Reply:
We cannot find the buttons by their real name otherwise. There is no toggle-camera property in the object returned by getAllButtons.

Reply:
Yeah, but there's a toggleButton

Reply:
Yes but if you look at what the code tried to do it was wrong because the button name (as per config.js) is not toggleButton, so you'd never find it and it just won't show up.